index.html

```diff
@@ -0,0 +1,190 @@
+    <div class="programs-wrapper">
+      <div class="program program1">
+        <img src="./images/program-outline/program01.png" alt="program01 icon">
+        <span> Lecture</span>
+        <p>Listen to the speakers from various countries about the message of sharing and opening</p>
+      </div>
+      <div class="program program2">
+        <img src="./images/program-outline/program02.png" alt="program02icon">
+        <span> CC Exibition</span>
+        <p>Listen to the speakers from various countries about the message of sharing and opening</p>
+      </div>
+      <div class="program program3">
+        <img src="./images/program-outline/program03.png" alt="Program03 icon">
+        <span>Forum</span>
+        <p>Listen to the speakers from various countries about the message of sharing and opening</p>
+      </div>
+      <div class="program program4">
+        <img src="./images/program-outline/program04.png" alt="Program04" icon">
+        <span>Workshop</span>
+        <p>Listen to the speakers from various countries about the message of sharing and opening</p>
+      </div>
+      <div class="program program5">
+        <img src="./images/program-outline/program05.png" alt="program05">
+        <span>CC Ingnite</span>
+        <p>Listen to the speakers from various countries about the message of sharing and opening</p>
+      </div>
+
+      <div class="join-btn-div">
+        <button type="button" class="btn join-btn">Join CC Global Summit 2015</button>
+      </div>
+    </div>
+
+  </section>
+
+  <!----Speakers section starts here-->
+  <section id="speakers" class="section-speakers section-wrapper">
+    <div class="speakers-container">
+      <h2 class="section-title speakers-section-title"> Featured Speakers </h2> 
+      <div class = "guide-bar speakers-guide-bar"></div>
+      <div class="card-container">
+        
+      </div>
+    </div>
+
+    <div class="see-more-speakers">
+      <a href="#">More </a>
+     <a href="#">  <i class="fa fa-chevron-down"></i> </a>
+    </div>
+  </section>
+
+  <section id="partner-id" class="section-partners section-wrapper">
+    <div class="partners-container">
+      <h2 class="section-title partners-section-title"> Partners</h2>
+      <div class="guide-bar partners-guide-bar"> </div>
+      <ul class="partners">
+        <li class="partner-item">
+          <a id="mozilla" href="#">mozilla</a>
+        </li>
+        <li class="partner-item">
+          <a id="google" href="#">Google</a>
+        </li>
+        <li class="partner-item">
+          <a id="naver" href="#">NAVER</a>
+        </li>
+        <li class="partner-item">
+          <a id="daumkakao" href="#">daumkakao</a>
+        </li>
+        <li class="partner-item partner-item-airbnb">
+          <a id="airbnb-icon"  href="#"> <img src="./images/airbnb_icon.png" alt=""> </a>
+          <a id="airbnb" href="#">airbnb</a>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <footer class="footer  section-wrapper">
```

Comment:
- [ ] Kindly make the text in the footer a little `bold` and use the given color swatches so as to match the design.

![image](https://user-images.githubusercontent.com/81932478/191731253-c1afa11c-a6ef-4a69-ba37-fc360fe4496a.png)